Dockerfile

```diff
@@ -105,7 +106,6 @@ libcairo2 \
 gfortran \
 r-base \
 time \
```

Comment:
There are new dependencies in the final stage of the build from the base image.  The content of some packages changed.  You should have eveything that PCAP-core has + the ones specific to ascatNgs:

https://github.com/cancerit/PCAP-core/blob/439ba182f796644240233775047991663c7e4d6e/Dockerfile#L78-L93

I think it's just 3:

https://github.com/cancerit/PCAP-core/compare/5.0.5...5.1.0#diff-3254677a7917c6c01f55212f86c57fbfR90-R93